jetty-websocket/javax-websocket-tests/src/test/java/org/eclipse/jetty/websocket/javax/tests/server/WebSocketServerContainerExecutorTest.java

```diff
@@ -205,6 +207,7 @@ public void testServerExecutor() throws Exception
         }
     }
 
+    @Disabled //TODO: the ContextHandler executor attribute is overwritten on ContextHandler.doStart() now we do attribute lookup lazily
```

Comment:
not sure how I should change this test or if I should just remove it

because we are now getting the executor attribute on the `ContextHandler` lazily in the `Supplier<WebSocketCoreClient>`, it is being overridden by the server executor in `ContextHandler.doStart()`

Reply:
I think it should be changed to check that the executor used by the container is the same as the one used by the server, and not a new executor. The test should be called `testServerExecutor`

Reply:
we already have the `testServerExecutor` test and it is just above this one, this one was just testing we could set the executor through the contextHandler attribute

Reply:
Ah OK.  So we can't set just an executor per context because the `doStart` overrides it.   But we can set a `HttpClient` or a `coreClient` per context.   I guess this is OK...